cmd/resources/serve.go

```diff
@@ -37,11 +38,20 @@ func (opts *serveOpts) setup(flags *pflag.FlagSet) {
 	flags.StringVar(&opts.Address, "address", "0.0.0.0", "Address to bind")
 	flags.IntVar(&opts.Port, "port", 8080, "Port on which the server will listen")
 	flags.StringArrayVarP(&opts.WatchPaths, "watch", "w", nil, "Paths to watch for changes")
+	flags.BoolVar(&opts.NoWatch, "no-watch", opts.NoWatch, "Do not watch for changes")
 	flags.StringVarP(&opts.Script, "script", "S", "", "Script to execute to generate a resource")
-	flags.StringVarP(&opts.ScriptFormat, "script-format", "f", "yaml", "Format of the data returned by the script")
+	flags.StringVarP(&opts.ScriptFormat, "script-format", "f", "json", "Format of the data returned by the script")
```

Comment:
Note: drive-by change. I set the default value to `json` to be consistent with JSON being the default format in other places as well.